style/collapseNavbar.css

```diff
@@ -0,0 +1,125 @@
+.collape-navbar-container {
+  display: none;
+}
+
+.overlay-container {
+  display: none;
+}
+
+@media screen and (min-width: 860px) {
+.menu-logo {
+  display: none
+}
+}
+
+@media screen and (max-width: 860px) {
+  .collape-navbar-container {
+    display: block;
+    position: absolute;
+    top: 0;
+    right: 0;
+    background-color: white;
+    width: 80%;
+    padding: 3rem;
+    z-index: 3;
+    height: 100%;
+  }
+
+  .header-navbar {
+    font-size: 2.5rem;
+    color: var(--primary-color);
+  }
+
+  .highlight-letter-header {
+    font-size: 4rem;
+  }
+
+  .overlay-container {
+    display: block;
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
```

Comment:
just use 0 once and it will set it to all directions, I think you don't need to write separately for every direction